layouts/default.vue

```diff
@@ -13,13 +11,17 @@
 import SbcFooter from 'sbc-common-components/src/components/SbcFooter.vue'
 import SbcHeader from 'sbc-common-components/src/components/SbcHeader.vue'
 import Breadcrumb from '~/components/Breadcrumb.vue'
+
 export default {
   components: {
     SbcFooter,
     SbcHeader,
     Breadcrumb,
   },
   middleware: ['commonHeaderSetup'],
+  created () {
+    console.log(`${this.$config.appName} v${this.$config.appVersion}`)
```

Comment:
Displays this:

![image](https://user-images.githubusercontent.com/10002889/148860986-7dcbe018-bafb-440f-83d5-eaef8b8fa409.png)

Reply:
cool